src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios.md

```diff
@@ -101,52 +97,83 @@ export const options = {
 };
 
 export default function () {
-  http.get('https://google.com/');
+  http.get('https://test.k6.io/');
 }
 ```
 
 </CodeGroup>
 
-Running the above script with `k6 run script.js`, additional information is added to the output as follows:
+If you run a script with scenarios, k6 output includes high-level information about each one.
+For example, if you run the preceding script, `k6 run scenario-example.js`,
+then k6 reports the scenarios as follows:
+
+<CodeGroup labels={["scenario example output"]} lineNumbers={[false]}>
+
+```bash
```

Comment:
```suggestion
```javascript
```

Reply:
@ppcano , just wondering, what's your preference for Javascript over bash to highlight text output? Originally, I was going to use `txt`, which would have rendered the text all white, then switched to bash since it highlights the values.

![image](https://user-images.githubusercontent.com/47385188/205880536-4a32f167-7188-4672-9bd3-bd9e8e7abc0a.png)

Reply:
@MattDodsonEnglish sorry. This should be `bash`. My suggestion was for `scenario-example.js`

Reply:
Oh that's a good catch anyway, thanks! Fixed with: https://github.com/grafana/k6-docs/commit/bd01aed7777877f5dddf32fb838b624184620c9f